src/index.ts

```diff
@@ -61,6 +61,12 @@ export interface CheckoutOptions {
   testing?: boolean
   /** Wether the Checkout should be a popup or an inline element  */
   display?: string
+  /** Wether the Checkout should have the customer details hidden */
```

Comment:
`customer details form`, so it's clearer. The details themselves are shown in a summary box.